libbeat/etc/kibana/dashboard.json.j2

```diff
@@ -0,0 +1,13 @@
+{
+  "hits": 0,
+  "timeRestore": false,
+  "description": "",
+  "title": "{{beatname}} Dashboard",
```

Comment:
I hoped the base structure here would not change too often. What is your experience from the last months? Did it change during minor releases?